synapse/handlers/typing.py

```diff
@@ -430,6 +431,27 @@ def _make_event_for(self, room_id):
             "content": {"user_ids": list(typing)},
         }
 
+    async def get_new_events_as(
+        self, from_key: int, service: ApplicationService, **kwargs
```

Comment:
Doesn't look like these new methods need to accept `kwargs`?

Can you add docstrings to them? 😄